pallene/ast.lua

```diff
@@ -83,7 +83,15 @@ declare_type("Field", {
 -- note: the following functions are why we need `if type(conss) == "table"` in parser.lua
 --
 
--- Return the variables names declared by a given toplevel node
+-- Returns a sequence containing the variable names declared by the specified
+-- toplevel node.
```

Comment:
It looks like your text editor is configured with lines of length 80?

Reply:
(Adding editorconfig support to your editor should take care of this)

Reply:
I manually formatted it to 80 columns. I'll add the editorconfig support ASAP.